app/helpers/inquirer.js

```diff
@@ -85,13 +86,39 @@ async function createNewTask(taskController, user) {
     const taskDetails = await inquirer.prompt([askDetails])
     const ca = await inquirer.prompt([confirmAction])
 
-    if(ca.confirm){
+    if (ca.confirm) {
         await taskController.create(new Task(null, taskTitle.title, taskDetails.details, 'pending', new Date().toISOString(), null, null, user.name))
         console.log('Task created')
-    }else {
+    } else {
         console.log('Cancelled task creation')
     }
 }
 
+async function showPendingTasks(taskController, user) {
+    let counter = 0;
+    let exit = false;
+    const tasksArray = await taskController.getPendingTasks();
+    const tasksByUser = [];
```

Comment:
move taskByUser to a new function. We gonna reuse it in the future. Something like getTasksByUser(tasks,user)..{return a new task array}